_dprhtml/js/cped.test.js

```diff
@@ -0,0 +1,30 @@
+import "../../en/cped";
+
+import { getDefinition, getGrammar, getInflectionGroup } from "./cped";
+
+describe("getDefinition", () => {
```

Comment:
Happy to add more test cases for any of these helper methods, but given the simplicity of the API I thought just a couple test cases would be sufficient.

Reply:
yeah please use your judgement. no tests at times to get over humps is also fine.